client/reader/following/main.jsx

```diff
@@ -40,11 +42,26 @@ const FollowingStream = props => {
 			] )
 		);
 	const placeholderText = getSearchPlaceholderText();
+	const userInUSA = getCurrentUserCountryCode( state ) === 'US';
```

Comment:
Hmmm, so this gives me a linter error because state is not defined, but the code works for some reason, and I think to do something like this.state I need to significantly change the FollowingStream in magical ways that I don't understand. And HALP.

Reply:
halping.